packages/components/src/styles/_typography.scss

```
@@ -0,0 +1,5 @@
+
+$a8c-fontFamily-sans: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell',
```

According to [CSS/Sass guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#general-syntax-and-writing-rules), camelCase should be avoided in variables.

> Use hyphens, not underscores or camelCase, when naming things like IDs, classes, variable names, mixins, placeholders. Good: `.site-title`, Bad: `.siteTitle` or `.site_title`.

```suggestion
$a8c-font-family-sans: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell',
```

---

I agree with @sirreal -- mixing two case conventions in one name is unpleasant to look at 🙂